internal/tiltfile/tiltfile_state.go

```diff
@@ -602,6 +637,40 @@ func (s *tiltfileState) assembleK8sV2() error {
 		return err
 	}
 
+	resourcedEntities := []k8s.K8sEntity{}
+	for _, r := range s.k8sByName {
+		resourcedEntities = append(resourcedEntities, r.entities...)
+	}
+
+	allEntities := append(resourcedEntities, s.k8sUnresourced...)
+
+	fragmentsToEntities := k8s.FragmentsToEntities(allEntities)
```

Comment:
as discussed, maybe delete this whole block since it's only used for one error message